Default.sublime-commands

```diff
@@ -0,0 +1,19 @@
+[
+	{
+		"caption": "KSP (Kontakt Script Processor): Compile",
```

Comment:
Let's shorten "KSP (Kontakt Script Processor)" to just "SublimeKSP" across this whole file.